### PR TITLE
merged MONDO:0035547 into MONDO:0014597

### LINKED ISSUE #7513: Predisposition to severe viral infection due to IRF7 deficiency and Immunodeficiency 39
Predisposition to severe viral infection due to IRF7 deficiency MONDO:0035547 and Immunodeficiency 39 MONDO:0014597 seem to be synonyms.  Please review.

Thank you,

Gioconda Alyea

### PR DESCRIPTION
**Pull request by sabrinatoro**
Closes #7513

### CODE REVIEW

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -337028,21 +337028,31 @@ id: MONDO:0014597
...
 synonym: "IMD39" RELATED ABBREVIATION [MONDO:Lexical, OMIM:616345]
 synonym: "immunodeficiency 39" EXACT [MONDO:Lexical, OMIM:616345]
 synonym: "immunodeficiency type 39" EXACT [MONDORULE:2, OMIM:616345]
 synonym: "IRF7 primary immunodeficiency disease" EXACT [MONDO:design_pattern, MONDO:patterns/disease_series_by_gene]
+synonym: "predisposition to severe viral infection due to IRF7 deficiency" EXACT [MONDO:0035547]
 synonym: "primary immunodeficiency disease caused by mutation in IRF7" EXACT [MONDO:design_pattern]
 xref: DOID:0111969 {source="MONDO:equivalentTo"}
+xref: ICD10CM:D84.8 {source="MONDO:mondoIsNarrowerThanSource", source="Orphanet:574918"}
+xref: MEDGEN:1830120 {source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
 xref: MEDGEN:904167 {source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:337044]: should one of these be preferredExternal?

File: src/ontology/mondo-edit.obo [hunk 2]
```diff
@@ -337028,21 +337028,31 @@ id: MONDO:0014597
...
 xref: OMIM:616345 {source="MONDO:equivalentTo"}
+xref: Orphanet:574918 {source="MONDO:equivalentTo"}
+xref: SCTID:1269234000 {source="MONDO:equivalentTo"}
 xref: UMLS:C4225358 {source="MONDO:equivalentTo", source="MONDO:MEDGEN", source="MEDGEN:904167"}
+xref: UMLS:C5681719 {source="MEDGEN:1830120", source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:337049]: same here, should one be prefereredExternal?

**Review by nicolevasilevsky — CHANGES_REQUESTED** [review]
minor changes requested

File: src/ontology/mondo-edit.obo (hunk 1 above)
Comment (sabrinatoro) [reply src/ontology/mondo-edit.obo:337044]: @nicolevasilevsky All the MEDGEN and UMLS cross-references are maintained by MedGen. Therefore anything we do with these x-refs will be automatically updated (and probably lost) during the next refresh. Also, they will be the one deciding what to do (e.g merge their own record). 
For these reasons, I did not add a "preferredExternal".

**Comment by sabrinatoro** [comment]
@nicolevasilevsky please see my response to your comments. thank you!

**Review by nicolevasilevsky — APPROVED** [review]
sounds good!

**Pull request merged**